### ISSUE TITLE
Modelling continuous outcomes that are not Normally distributed

### ISSUE THREAD

**Issue by joannadiong** [CausalInference/gfoRmula#34]
Hi Sean,

I understand that the R g-formula package currently only supports linear regression for continuous outcomes, which may create issues if the outcome isn't Normally distributed, potentially such as the outcome I'm trying to model using this package.

At present, I understand there are updates being made to the SAS g-formula macro to support other options for modelling continuous outcome (e.g., Poisson, zero-inflated normal, etc.) but we're not sure on the status in terms of similar updates being made to the R g-formula package. 

Would you have suggestions on how to proceed in this case? 
E.g. transforming the outcome data and modelling that (but this comes with disadvantages), or a form of robust regression, or await package updates?

Many thanks,
Jo

**Comment by stmcg** [comment]
Hi Jo,

That's right, the R package currently only supports linear regression for continuous outcomes. 

Just to clarify, this doesn't necessarily require assuming the outcome is normally distributed. Rather we just require the model for the conditional outcome mean to be correctly specified. In any case, I can see scenarios where different types of outcome models would be helpful.

I don't believe there is any current / near future work on updating the R package for other types of outcome models. So I wouldn't wait on package updates for your analyses. 

It's hard to give more concrete advice with how to proceed in your analysis without more context. If you'd like to elaborate on your analysis question, feel free to reach out at sean_mcgrath@g.harvard.edu (suggesting taking this discussion to email since it isn't on package bugs). My time is a bit limited at the moment, but I should be able to take a look and give some pointers

Best,
Sean
